code/edgemedia/src/main/java/com/adobe/marketing/mobile/edge/media/internal/MediaRealTimeSession.kt

```diff
@@ -0,0 +1,195 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.edge.media.internal
+
+import androidx.annotation.VisibleForTesting
+import com.adobe.marketing.mobile.Event
+import com.adobe.marketing.mobile.EventSource
+import com.adobe.marketing.mobile.EventType
+import com.adobe.marketing.mobile.edge.media.internal.xdm.XDMMediaEvent
+import com.adobe.marketing.mobile.edge.media.internal.xdm.XDMMediaEventType
+import com.adobe.marketing.mobile.services.Log
+import com.adobe.marketing.mobile.util.StringUtils
+
+internal class MediaRealTimeSession(
+    id: String,
+    state: MediaState,
+    dispatchHandler: (Event) -> Unit
+) : MediaSession(id, state, dispatchHandler) {
+
+    companion object {
+        private const val LOG_TAG = MediaInternalConstants.LOG_TAG
```

Comment:
can we use `typealias` instead of new constant?

Reply:
A typealias will rename a type, not the value so that wouldn't work here. However, I can just import `MediaInternalConstants.LOG_TAG` here and use it instead of creating a new variable.